bftengine/src/bftengine/ReplicaImp.hpp

```diff
@@ -463,7 +463,8 @@ class ReplicaImp : public InternalReplicaApi, public ReplicaForStateTransfer {
                                         {"onSeqNumIsStable", onSeqNumIsStable},
                                         {"onTransferringCompleteImp", onTransferringCompleteImp},
                                         {"consensus", consensus},
-                                        {"timeInActiveView", timeInActiveView}});
+                                        {"timeInActiveView", timeInActiveView},
+                                        {"checkpointFromCreationToStable", checkpoint_creation_to_stable}});
```

Comment:
It looks like you missed adding `timeInStateTransfer` here.

Reply:
fixed